packages/server/api/routes/reviews.router.js

```diff
@@ -0,0 +1,44 @@
+const express = require('express');
+
+const router = express.Router({ mergeParams: true });
+
+// controllers
+const reviewsController = require('../controllers/reviews.controller');
+
+/**
+ * @swagger
+ * /reviews:
+ *  get:
+ *    tags:
+ *    - reviews
+ *    summary: Get all reviews or reviews with specific ProductId
+ *    description:
+ *      Will return an array with all reviews as objects or will return all reviews for specific productId.
+ *    produces: application/json
+ *    parameters:
+ *     - in: query
+ *       name: productId
+ *       schema:
+ *         type: number
+ *         required: false
+ *       description: The product id of the reviews to get
+ *    responses:
+ *      200:
+ *        description: Successful request
+ *      5XX:
+ *        description: Unexpected error.
+ */
+router.get('/', (req, res, next) => {
+  if (req.query.productId) {
+    reviewsController
+      .getReviewsByProductId(req.query.productId)
+      .then((result) => res.json(result))
```

Comment:
Looks like if the `productId` query parameter is set, this route will attempt to send 2 responses. The first one here and the next one on line 40. We can't send 2 responses so the second call will fail and show something in the terminal. Can you try this out and fix?